README.md

```diff
@@ -4,6 +4,7 @@
 
 - **Version 2**
 
+  - 2.0.8: added ToolTip component.
```

Comment:
You've added the ToolTip _and_ the NodeToolTip components, but you're only exporting the ToolTip component (so right now, the NodeToolTip seems redundant).

However, do check out the other comment about maybe removing the `ToolTip` altogether and only using the `NodeToolTip`.